arvo/jael/examples.md

```diff
@@ -0,0 +1,311 @@
+                939.529.329.928[...truncated for brevity...]
+              ]
+            ]
+            [ p=2
+                q
+              [ crypto-suite=1
+                  pass
+                2.215.774.809.906.200.376.0[...truncated for brevity...]
+              ]
+            ]
+            [ p=3
+                q
+              [ crypto-suite=1
+                  pass
+                707.070.568.606.374.[...truncated for brevity...]
+              ]
+            ]
+          }
+          sponsor=~
+        ]
+      ]
+    }
+  ]
+]
+```
+
+Along with giving us the current information, Jael will also subscribe us to any future updates for the ships in question. Such updates will come as additional `%public-keys` `gift`s, but rather than a `%full` `public-keys-result`, they'll instead contain either a `%diff` or `%breach` `public-keys-result`, depending on what's happened to the ship in question. It's difficult to simulate such events for demonstrative purposes so an example is not included, but you can look at the [$public-keys-result](@/docs/arvo/jael/data-types.md#public-keys-result) to get an idea.
+
+Jael maintains a `(jug duct ship)` and its reverse `(jug ship duct)` in its state to track subscriptions. If we do a [subscriptions](@/docs/arvo/jael/scry.md#subscriptions) scry and filter the result for `~dopzod`, we can see the duct of our thread has now been added to the `~dopzod` `set`. Our thread does this, and will output something like:
+
+```
+{~[/gall/use/spider/0w1.vGVi-/~sampel-palnet/thread/~.dojo_0v6.0hlak.dam1b.bcdou.ai7gq.19fi8/sub-pubkeys /dill //term/1]}
+```
+
+At this point our thread will send a `%nuke` `task` like `[%nuke (silt ~[~dopzod])]` to cancel our subscription. Jael doesn't respond to it, but now, with our thread having finished and exited, we can again scry & filter for subscriptions to `~dopzod`:
+
+```
+> =/(a .^([yen=(jug duct ship) ney=(jug ship duct) nel=(set duct)] %j /=subscriptions=/1) (~(get ju ney.a) ~dopzod))
+~
+```
+
+As you can see the `set` is now empty, so we know the `%nuke` succeeded and Jael will no longer send us pubkey updates for `~dopzod`. One thing to note about `%nuke` is that it *must* come from the same `duct` as the original subscription. You can't unsubscribe another app, ship, thread or what have you, so if we'd tried `%nuke`ing the subscription from a separate thread it wouldn't have worked.
+
+# `%rekey`
+
+Here we'll look at changing a ship's private keys by sending a `%rekey` `task` to Jael. 
+
+Here's a thread that will scry for a ship's current `life`, increment it, generate a random private key from some entropy, and send a `%rekey` `task` to jael to increase the ship's `life` and set the new private key.
+
+This example was run on an offline **copy** of a real ship booted with `urbit -L` because you can't change the `life` of a `urbit -F` fake ship. Setting a random private key this way on a real ship is obviously a bad idea so please be careful not to do that.
```

Comment:
```suggestion
**This example was run on an offline _copy_ of a real ship booted with `urbit -L` because you can't change the `life` of a `urbit -F` fake ship. Setting a random private key this way on a real ship is obviously a bad idea so please be careful not to do that.**
```
This is dangerous enough that I think the whole paragraph should be bolded. I'd actually consider removing this example entirely. I'm not sure the educational value is worth the risk of someone needing to breach if they forget to use `-L`. Not sure if a breach would actually be needed, but either way it seems too easy to end up in a bad spot due to carelessness or a typo, for not much gain.